docs/View-Reference.md

```diff
@@ -18,6 +18,21 @@ view(type: ListView) {  // since 1.21
         names(String... jobNames)
         regex(String regex)
     }
+    jobFilters { // since 1.28
+        regex {
+            includeMatched() | includeUnmatched() | excludeMatched() | excludeUnmatched() // what to do with matching Jobs
+            type(String type) // what to match - one of 'NAME', 'DESCRIPTION', 'SCM', 'EMAIL', 'MAVEN', 'SCHEDULE', 'NODE'
```

Comment:
This is confusing. The UI calls this "Match Value", but you called it `type` and "Match Type" in the UI is one of the "Include Matched", "Exclude Matched" etc values. How about changing it  to `value` or `matchValue`?

And why is are there explict methods for the "Match Types", but not for the "Match Values"? I would prefer only one method for the "Match Type" as well, so it's more obvious that only one value can be selected.